core/test/financial_templates/PricelessPositionManager.js

```diff
@@ -467,5 +473,37 @@ contract("PricelessPositionManager", function(accounts) {
     await pricelessPositionManager.redeem({ rawValue: numTokens }, { from: sponsor });
     assert(await didContractThrow(pricelessPositionManager.redeem({ rawValue: numTokens }, { from: sponsor })));
   });
+
+  it("Basic fees", async function() {
+    // Set up position.
+    await collateral.approve(pricelessPositionManager.address, toWei("1000"), { from: sponsor });
```

Comment:
As mentioned above, general practice is to always use `increase/decreaseApproval` instead of `approve`. I know this is just a test, but just FYI

Reply:
Understood -- if we want to change this, we should probably do so in a separate PR since we use this method of approval in almost all of our tests.